<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>geometriesvisualization</class>
 <widget class="QDialog" name="geometriesvisualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voxelized geometry visualization</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>561</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="GenerateMap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Map</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Axis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the axis of the slices to be shown&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string>Choose the axis of the slice to be visualized</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeAxiss">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VoxelsDataFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddVoxelsDataFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Choose voxels ID data file</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDicom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CT</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonVoxIDs">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IDs</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSaveExt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxNumberZ">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of voxels on Z-axis</string>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxNumberY">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of voxels on Y-axis</string>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxNumberX">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Number of voxels on X-axis</string>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxSizeZ">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add voxel Z-size in mm</string>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxSizeY">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add voxel Y-size in mm</string>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReadVoxelsData">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Read voxels ID data file</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderSliceID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add slice ID</string>
    </property>
    <property name="statusTip">
     <string>Choose the slice ID to be visualized (in the chosen axis)</string>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxSizeX">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add voxel X-size in mm</string>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenResDir">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPET">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PET</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVal">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarFileReading">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxColorGradien">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Choose the color gradient method</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>9</y>
     <width>771</width>
     <height>561</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>gui/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
